src/components/remove-networked-object-button.js

```diff
@@ -14,6 +14,7 @@ AFRAME.registerComponent("remove-networked-object-button", {
       this.el.parentNode.setAttribute("visible", false);
 
       this.targetEl.addEventListener("animationcomplete", () => {
+        NAF.utils.takeOwnership(this.targetEl);
```

Comment:
I guess you probably already know this, but `takeOwnership` does not always guarantee ownership after it is called.

Reply:
yep, I considered that -- it seems right to remove it anyway if it fails for this case since at that point the object has scaled down to be tiny